tools/executors/playwright/playwright.impl.ts

```diff
@@ -0,0 +1,141 @@
+/**
+ * @license
+ * Copyright 2021 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import 'dotenv/config';
+import { join } from 'path';
+import {
+  ExecutorContext,
+  logger,
+  parseTargetString,
+  readTargetOptions,
+  runExecutor,
+} from '@nrwl/devkit';
+import { default as runCommandsExecutor } from '@nrwl/workspace/src/executors/run-commands/run-commands.impl';
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export type Json = { [k: string]: any };
+
+export interface PlaywrightExecutorOptions extends Json {
+  playwrightConfig: string;
+  watch?: boolean;
+  tsConfig?: string;
+  devServerTarget?: string;
+  headed?: boolean;
+  headless?: boolean;
+}
+
+export default async function cypressExecutor(
```

Comment:
fix: rename to `playwrightExecutor`. Adjust the function name in other files if necessary.

```suggestion
export default async function playwrightExecutor(
```